Makefile

```diff
@@ -1,7 +1,7 @@
 SHELL=/bin/bash
 
-export KUBEVIRT_VERSION = main
-export KUBEVIRT_TAG = main
+export KUBEVIRT_VERSION = v1.4.0-rc.0
+export KUBEVIRT_TAG = release-1.4
```

Comment:
FWIW `_VERSION` and `_TAG` are bad names IMHO but I'll rework this in a separate PR on main.